packages/teams-js/src/internal/communication.ts

```diff
@@ -60,10 +60,14 @@ export function initializeCommunication(validMessageOrigins: string[] | undefine
   }
 
   if (!Communication.parentWindow) {
-    GlobalVars.isFramelessWindow = true;
-    /* eslint-disable  @typescript-eslint/ban-ts-comment */
-    // @ts-ignore: window as ExtendedWindow
-    (window as ExtendedWindow).onNativeMessage = handleParentMessage;
+    const extendedWindow = (Communication.currentWindow as unknown) as ExtendedWindow;
```

Comment:
Can be directly casted to ExtendedWindow

Reply:
It can't. TypeScript complains. Which is why there was a `@ts-ignore` comment here before. I think going through `as unknown` and not trying to disable type checks is better, which is why I changed this. The behavior should be the same.